release.json

```diff
@@ -45,8 +45,8 @@
         "WINDOWS_APMINJECT_SHASUM": "c5c228f6030ce2e19b8bfb28bd054dc246c7f1c799431e73018e6cb46ee59d2f"
     },
     "release-a6": {
-        "INTEGRATIONS_CORE_VERSION": "7.49.0-rc.2",
-        "OMNIBUS_SOFTWARE_VERSION": "7.49.0-rc.1",
+        "INTEGRATIONS_CORE_VERSION": "f81ad6b582218dd9bca47bd78ddd6d871613540b",
```

Comment:
this is the latest commit on the postgres integration, which contains the bump to the latest version of psycopg2